### DISCUSSION TITLE
How to add the Shadow colour to bottom of Header in navigation 6

### DISCUSSION THREAD

**Discussion by gkasireddy202** [react-navigation/react-navigation#11991] [Q&A]
I need to add the shadow color to the bottom of the header like in the screenshot.
I highlighted the header shadow color with green in the screenshot.
![IMG_3485](https://github.com/react-navigation/react-navigation/assets/90603484/1fa5cb07-bdcb-41e2-9a8e-3761e5ea1251)

**Comment by alordiel** [comment]
In your `screenOptions` you should add:
```javascript
headerStyle: {
          elevation: 9,
          shadowOpacity:10,
          borderBottomWidth:1,
},
 ``` 
 And check if you have `headerShadowVisible: true,`  ([docs](https://reactnavigation.org/docs/elements/#headershadowvisible))
